i18n/locales/en-US.json

```diff
@@ -48,6 +48,10 @@
                     "label": "Select your preferred language",
                     "info": "Please note that in order for this change to take effect on the whole application you'll need to restart it once."
                 },
+                "keybind-change": {
+                    "label": "Change the Vanish Keybind",
+                    "info": "Control used for CTRL on Windows/Linux, Command for the MacOS commonly used Keybind CMD. For Example: Alt+CommandOrControl+I will be Alt+CTRL+I on Win/Linux and Alt+CMD+I on MacOS. Further Examples: CommandOrControl+I, Control+Alt+F9, Command+Alt+Shift+I."
+                },
```

Comment:
i think it would make sense to either add `\n` line breaks or move the info translation to an array and loop through it in the template

Reply:
same as de-DE